src/Serialization/containers.jl

```diff
@@ -542,7 +631,7 @@ function save_object(s::SerializerState, obj::SRow)
   end
 end
 
-function load_object(s::DeserializerState, ::Type{<:SRow}, params::NCRing)
+function load_object(s::DeserializerState, ::Type{<:SRow}, params::U) where U
```

Comment:
what is this change needed for? SRows can only contain NCRingElems (and nothing else), so I don't really understand this.

If this is indeed needed for something, you still don't use the `U` anywhere, so it is better to just not introduce the type var at all
```suggestion
function load_object(s::DeserializerState, ::Type{<:SRow}, params)
```